projects/plugins/jetpack/extensions/extended-blocks/index.scss

```diff
@@ -0,0 +1,28 @@
+.is-beta-extension {
```

Comment:
Maybe we can use `@automattic/jetpack-base-styles` for the colors here?

Reply:
I've moved the code straight into the edtior.js file. I think it's a good place to put the code. c68ac30907698a2fe934eb62751e9870f7287c4c